code/modules/unit_tests/_unit_tests.dm

```diff
@@ -31,6 +30,7 @@
 #include "emoting.dm"
 #include "keybinding_init.dm"
 #include "machine_disassembly.dm"
+#include "ntnetwork_tests.dm"
```

Comment:
Still not alphabetical order 😛

Reply:
Why'd you resolve this? :smile:

Reply:
I thought I did? one sec

Reply:
It should be after metabolizing

Reply:
wait a seic it is?  Its whoding up in order for me?

Reply:
![image](https://user-images.githubusercontent.com/35135081/97927868-b05a5f80-1d1a-11eb-9cc0-b5faae5e8670.png)

You must not have committed/pushed it then.

Reply:
wait a sec, isn't it n after m?  Did I put it in the wrong file?

Reply:
ok I am retarded

Reply:
I kept thinking, "But n is after m?"  Then I see the other two files uggh